src/box/port.h

```diff
@@ -96,6 +96,8 @@ struct port_lua {
 	int ref;
 	/** Number of entries dumped to the port. */
 	int size;
+	/** Bottom index of values to be dumped. */
+	int bottom;
```

Comment:
I think that bottom (or count) should be passed explicitly to `port_lua_create`.

Reply:
Done (introduced `port_lua_create_with_bottom`).